contrib/kubespray/script/openpai-generator.py

```diff
@@ -188,8 +227,8 @@ def hived_config_prepare(worker_dict, node_resource_dict):
             logger.error("Allocatable GPU number in {0} is 0, current quick start script does not allow.".format(key))
             logger.error("Please remove {0} from your workerlist, or check if the device plugin is running healthy on the node.".format(key))
             sys.exit(1)
-        min_cpu = min(min_cpu, node_resource_dict[key]["cpu-resource"])
-        min_mem = min(min_mem, node_resource_dict[key]["mem-resource"])
+        min_cpu = min(min_cpu, node_resource_dict[key]["cpu-resource"] - pai_daemon_resource_dict["cpu-resource"])
```

Comment:
Could we leave a buffer (such as reserve 1CPU 1GB?), and also minus it here? So that even if the whole caluculate is not precise, it will not cause previous issue.

Reply:
Fxied